tests/decode/test1_suit_old_formats/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@
 
 cmake_minimum_required(VERSION 3.13.1)
 
+set(PROJECT_NAME test1_suit_old_formats)
```

Comment:
Could we grab this in the `test_template.cmake` by using the parent dir of the current cmakelist.txt ? Maybe that is a bit too "magic".

Reply:
I think it's good that this is properly CTRL-F-able.